coderd/coderd_test.go

```diff
@@ -413,7 +410,14 @@ func TestAuthorizeAllEndpoints(t *testing.T) {
 
 			if !routeAssertions.NoAuthorize {
 				assert.NotNil(t, authorizer.Called, "authorizer expected")
-				assert.Equal(t, routeAssertions.StatusCode, resp.StatusCode, "expect unauthorized")
+				if routeAssertions.StatusCode != 0 {
+					assert.Equal(t, routeAssertions.StatusCode, resp.StatusCode, "expect unauthorized")
+				} else {
+					// It's either a 404 or 403.
```

Comment:
feels like a cop-out.  Can we at least get an issue to track going back and fixing up what we should be getting?

Reply:
https://github.com/coder/coder/issues/2296